ethernet-view/src/components/ChartMenu/ChartList/ChartElement/ChartElement.tsx

```diff
@@ -1,10 +1,11 @@
 // @ts-ignore
 import CanvasJSReact from '@canvasjs/react-charts';
-import { ChartId, DataSeries, MeasurementId, MeasurementInfo, Point } from "components/ChartMenu/types";
+import { ChartId, DataSeries, MeasurementId, MeasurementInfo } from "components/ChartMenu/types";
 import styles from "./ChartElement.module.scss";
 import { AiOutlineCloseCircle } from 'react-icons/ai'
 import { MutableRefObject, memo, useCallback, useRef, DragEvent, useEffect } from "react";
 import { useInterval } from 'common';
+import { createDataSeries } from 'components/ChartMenu/utils';
```

Comment:
I don't like the idea of an `utils` file, I would put it next to the code it is used, even if you export it so it can be used somewhere else